canonical-websites/www.ubuntu.com
# Fix link on download menu

## Done

Fix link in download menu

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- See that only the word ubuntu is linked in the sentence 'Create a bootable USB stick on Ubuntu'

## Issue / Card

Fixes [#697](https://github.com/ubuntudesign/web-squad/issues/697)
